cmd/temporalite/main.go

```diff
@@ -289,3 +311,21 @@ func getPragmaMap(input []string) (map[string]string, error) {
 	}
 	return result, nil
 }
+
+func getDynamicConfigValues(input []string) (map[dynamicconfig.Key][]dynamicconfig.ConstrainedValue, error) {
+	ret := make(map[dynamicconfig.Key][]dynamicconfig.ConstrainedValue, len(input))
+	for _, keyValStr := range input {
+		keyVal := strings.SplitN(keyValStr, "=", 2)
+		if len(keyVal) != 2 {
+			return nil, fmt.Errorf("dynamic config value not in KEY=JSON_VAL format")
+		}
+		key := dynamicconfig.Key(keyVal[0])
+		// We don't support constraints currently
```

Comment:
Constraints sound nice... what are they and is there future work we should track to support them?

Reply:
I don't really know, I'd have to speak to the server team. My use case and most use cases I've seen don't have these constraints. We can block this PR for that if we really need to.